src/client/apiclient.ts

```diff
@@ -21,10 +21,10 @@ export const getCompanies =
     };
 
 export const getDissolvedCompanies =
-async (apiKey: string, companyName: string, requestId, searchType: string): Promise<DissolvedCompaniesResource> => {
+async (apiKey: string, companyName: string, requestId, searchType: string, page: number): Promise<DissolvedCompaniesResource> => {
     const api = createApiClient(apiKey, undefined, API_URL);
     const companiesResource: Resource<DissolvedCompaniesResource> =
-        await api.dissolvedSearch.getCompanies(companyName, requestId, searchType);
+        await api.dissolvedSearch.getCompanies(companyName, requestId, searchType, page * DISSOLVED_SEARCH_NUMBER_OF_RESULTS - 20);
```

Comment:
I'd be tempted to define the calculation here as a constant - also - is the `-20` just `- DISSOLVED_SEARCH_NUMBER_OF_RESULTS`?